omnigibson/action_primitives/curobo.py

```diff
@@ -0,0 +1,590 @@
+# Third Party
```

Comment:
How much of this file is directly third party vs written / modified by you? Should we credit via URL here?

Reply:
oh LOOOL the entire file was written by me. That comment was about importing third party modules before the main modules, but the rest of the comment got cut off haha

Reply:
Lol I see. Thanks!